src/Stratis.Bitcoin.Features.MemoryPool/Rules/CheckTxOutDustRule.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Microsoft.Extensions.Logging;
+using NBitcoin;
+using NBitcoin.Policy;
+using Stratis.Bitcoin.Features.MemoryPool.Interfaces;
+
+namespace Stratis.Bitcoin.Features.MemoryPool.Rules
+{
+    /// <summary>
+    /// This rule ensures that the transaction does not contain any outputs which are considered dust.
+    /// </summary>
+    public sealed class CheckTxOutDustRule : MempoolRule
+    {
+        public CheckTxOutDustRule(
+            Network network,
+            ITxMempool mempool,
```

Comment:
ITxMempool is not needed anymore (not  a big deal though).

Reply:
Its needed by the other mempool rules ;)